cfpb/consumerfinance.gov
# Related posts: Add aria label to "View more" link

## Changes

- Related posts: Add aria label to "View more" link and associated translations.


## How to test this PR

1. Pages with a related posts module in the sidebar, like /consumer-tools/, when you inspect the "View more" link should have an aria-label attribute on the link.
